docs/overview.md

```diff
@@ -0,0 +1,134 @@
+primitives. Managed resources with an API version of `v1beta1` or higher support
+every field that the cloud provider does for the given resource. You can find
+the Managed Resources and their API specifications for each provider on
+[doc.crds.dev] and learn more in the [managed resource documentation]. 
+
+## Composition
+
+Composition refers the machinery that allows you to bundle managed resources
+into higher-level infrastructure units, using only the Kubernetes API. New
+infrastructure units are defined using the `InfrastructureDefinition`,
+`InfrastructurePublication`, and `Composition` types, which result in the
+creation of new CRDs in a cluster. Creating instances of these new CRDs result
+in the creation of one or more managed resources. You can learn more about all
+of these concepts in the [composition documentation].
+
+## OAM
+
+Crossplane supports application management as the Kubernetes implementation of
+the [Open Application Model]. As such, Crossplane currently implements the
+following OAM API types as Kubernetes custom resources.
+
+* `WorkloadDefinition`: defines the kind of components that an application
+  developer can use in an application, along with the component's schema.
+  * Crossplane also implements the core `ContainerizedWorkload` type.
+    Infrastructure owners may define any resource as a workload type by
+    referencing it in a `WorkloadDefinition`.
+* `Component`: describe functional units that may be instantiated as part of a
+  larger distributed application. For example, each micro-service in an
+  application is described as a `Component`.
+* `Trait`: a discretionary runtime overlay that augments a component workload
+  type with additional features. It represents an opportunity for those in the
+  application operator role to make specific decisions about the configuration
+  of components, without having to involve the developer.
+  * Crossplane also implements the core `ManualScalerTrait` type.
+* `ApplicationConfiguration`: includes one or more component instances, each
+  represented by a component definition that defines how an instance of a
+  component spec should be deployed.
+
+For more information, take a look at the [OAM documentation].
+
+<!-- Named Links -->
+
+[Crossplane Concepts]: crossplane-concepts.png
+[providers]: #providers
+[provider-azure]: https://github.com/crossplane/provider-azure
+[packages]: #packages
+[managed resources]: #managed-resources
+[OAM]: #oam
+[CRDs]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
+[controllers]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-controllers
+[providers documentation]: providers.md
+[doc.crds.dev]: https://doc.crds.dev
+[managed resources documentation]: managed-resources.md
+[composition documentation]: composition.md
+[Open Application Model]: https://oam.dev/
+[OAM documentation]: oam.md
```

Comment:
Dependent on @negz naming of this doc

Reply:
is this needed before merge @hasheddan @prasek?

Reply:
I think we can go ahead and update if it doesn’t match 👍